services/system/Accounts/ui/src/AccountSelection.tsx

```diff
@@ -163,17 +170,23 @@ export const AccountSelection = () => {
   const disableModalSubmit: boolean =
     accountStatus !== (isInvite ? "Available" : "Taken");
 
-  const onAccountSelection = (accountId: string) => {
+  // AccountsList only shows accounts already connected to apps,
+  // so all that's needed here is a "direct" login.
```

Comment:
The accounts list shows *all* user accounts that have been imported to this device. The reason `loginDirect` is used is because it's the name of the function used to login to a third-party app (`accounts:plugin->login` can't be used because that only ever means logging into the caller app).